src/main/java/org/intellij/sdk/codesync/codeSyncSetup/CodeSyncSetup.java

```diff
@@ -633,13 +633,6 @@ public static void saveIamUser(String email, String iamAccessKey, String iamSecr
         }
 
         userFile.setActiveUser(email, iamAccessKey, iamSecretKey);
-        try {
```

Comment:
Maybe we can catch DB error here instead? In case user query fails for some reason